Naming/src/main/java/com/epam/engx/cleancode/naming/task5/FileManager.java

```diff
@@ -42,43 +42,43 @@ private boolean isInvalidFileType(String fileName) {
     }
 
     private boolean isInvalidImage(String fileName) {
-        FileExtPred imageExtensionsPredicate = new FileExtPred(TYPES);
+        FileExtensionsPredicate imageExtensionsPredicate = new FileExtensionsPredicate(IMAGES_TYPES);
         return !imageExtensionsPredicate.test(fileName);
     }
 
     private boolean isInvalidDocument(String fileName) {
-        FileExtPred documentExtensionsPredicate = new FileExtPred(TYPES2);
+        FileExtensionsPredicate documentExtensionsPredicate = new FileExtensionsPredicate(DOCUMENTS_TYPES);
         return !documentExtensionsPredicate.test(fileName);
     }
 
     private List<String> files(String directoryPath, String[] allowedExtensions) {
```

Comment:
```suggestion
    private List<String> listFiles(String directoryPath, String[] allowedExtensions) {
```
it similar with other methods